build.ps1

```diff
@@ -59,14 +61,14 @@ if ($RealSign) {
     $signType = 'real'
 }
 
-$commonBuildArgs = echo $RepoRoot\build\build.proj /m /nologo /p:Configuration=$Configuration /p:Platform=$Platform /p:SignType=$signType 
+$commonBuildArgs = echo $RepoRoot\build\build.proj /m /nologo /p:Configuration=$Configuration /p:Platform=$Platform /p:SignType=$signType /p:Verbosity=$Verbosity
```

Comment:
Further, all of the loggers mentioned are file loggers, so you're just getting default console verbosity output. I think you're looking for `/verbosity:minimal` or `/consoleloggerparameters:Verbosity=minimal`.